From dff05360665d948914458f19f92f245397b4bc6b Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nwani@tower-research.com>
Date: Mon, 22 Jan 2018 12:41:11 +0000
Subject: [PATCH] Add declaration before using method

Source: chromium-browser_61.0.3163.100-1~deb9u1.debian.tar.xz
description: forward declare method that is defined after it is used
author: Michael Gilbert <mgilbert@debian.org>
---
 .../src/3rdparty/chromium/third_party/WebKit/Source/wtf/LinkedHashSet.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/qtwebengine/src/3rdparty/chromium/third_party/WebKit/Source/wtf/LinkedHashSet.h b/qtwebengine/src/3rdparty/chromium/third_party/WebKit/Source/wtf/LinkedHashSet.h
index 4f6feed..0235c14 100644
--- a/qtwebengine/src/3rdparty/chromium/third_party/WebKit/Source/wtf/LinkedHashSet.h
+++ b/qtwebengine/src/3rdparty/chromium/third_party/WebKit/Source/wtf/LinkedHashSet.h
@@ -496,6 +496,8 @@ inline LinkedHashSet<T, U, V, W>& LinkedHashSet<T, U, V, W>::operator=(const Lin
     return *this;
 }
 
+inline void swapAnchor(LinkedHashSetNodeBase& a, LinkedHashSetNodeBase& b);
+
 template<typename T, typename U, typename V, typename W>
 inline void LinkedHashSet<T, U, V, W>::swap(LinkedHashSet& other)
 {
-- 
2.15.0

